src/templates/home.html

```diff
@@ -0,0 +1,23 @@
+{% extends '_base.html' %}
+
+{% block title %}Home{% endblock %}
+
+{% block content %}
+<div style="text-align: center;">
+  {% if user.is_authenticated %}
+    <p>Welcome, {{ user.email }}!</p>
+    <img src="https://media.tenor.com/WJrxtr--PCUAAAAi/workout-gym.gif" alt="Workout GIF">

```

Comment:
pobierz te grafiki do static i serwuj je od siebie, oduzależni Cie to od dostępności zewnętrzynch stronek i przyspieszy działanie apki